WcaOnRails/spec/features/sign_up_spec.rb

```diff
@@ -82,6 +83,75 @@
 
       expect(page).to have_selector('#have-competed', visible: true)
     end
+
+    it "remembers their selected wca id on validation error" do
+      visit "/users/sign_up"
+      click_on "I have competed in a WCA competition."
+      # They have not selected a valid WCA ID yet, so don't show the birthdate verification
+      # field.
+      expect(page.find("div.user_dob_verification", visible: false).visible?).to eq false
+
+      selectize_input = page.find("div.user_unconfirmed_wca_id .selectize-control input")
+      selectize_input.native.send_key(person.wca_id)
+      # Wait for selectize popup to appear.
+      expect(page).to have_selector("div.selectize-dropdown", visible: true)
+      # Select item with selectize.
+      page.find("div.user_unconfirmed_wca_id input").native.send_key(:return)
```

Comment:
Ah, good call! I've made this change everywhere. Too much copy paste going on here =(